scripts/qa_crowbarsetup.sh

```diff
@@ -3698,6 +3698,9 @@ function oncontroller_testsetup
 {
     . .openrc
     oncontroller_prepare_functional_tests
+    if [[ $hacloud = 1 ]] ; then
+        crm_mon --failcounts -1 | grep "fail-count=" && complain 34 "Cluster resources' failures detected"
```

Comment:
I'm not sure that's right.  At least it doesn't catch failures like this:

```
Failed Actions:
* nova-compute_monitor_10000 on remote-d52-54-77-77-77-03 'not running' (7): call=13560, status=complete, exitreason='none',
    last-rc-change='Thu Feb  2 17:08:13 2017', queued=0ms, exec=0ms
```

Please can you update the commit message to explain which types of errors it *does* catch, so we can compare the two and decide whether it's also important to catch this one?